pkgs/development/tools/documentation/doxyrest/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchgit, cmake, lib, expat, ragelDev, which, lua5 }:
+
+stdenv.mkDerivation rec {
+  name = "doxyrest";
+  # Somehow fetchFromGithub was ignoring submodules
+  src = fetchgit {
+    url = "https://github.com/vovkos/doxyrest_b";
+    rev = "3500bc0101c5ebe4921d8cca49b42e209e761971";
```

Comment:
Why this release won't do it:

https://github.com/vovkos/doxyrest_b/releases/tag/doxyrest-2.0.0

?

Reply:
It's much too old. A lot of fixes have been put in master since then.

Reply:
To be clearer, the current setup corresponds to https://github.com/vovkos/doxyrest/releases/tag/doxyrest-2.1.0

It's just that the build repo doesn't release the same way (since it is updated with each push).